README.md

```diff
@@ -186,13 +170,11 @@ kdk ssh
 kdk destroy
 ```
 
-
 ## Customization
 
 * **NOTE:**  By default, the `KDK` uses a set of opinionated dotfiles. 
 To customize, fork [this](https://github.com/cisco-sso/yadm-dotfiles) repo, make changes, and update `~/.kdk/config.yaml` to reference customized fork.
```

Comment:
It's no longer ~/.kdk/config.yaml but `~/.kdk/<kdk-name>/config.yaml`

Reply:
fixed